resource/tasks.json

```diff
@@ -718,12 +718,12 @@
         ],
         "maxTimes": 3,
         "next": [
-            "AdverseConfirm",
+            "RaidConfirm",
             "#self"
         ]
     },
-    "AdverseConfirm": {
-        "baseTask": "ChangeToAdverseDifficulty",
+    "RaidConfirm": {
+        "baseTask": "ChangeToRaidDifficulty",
```

Comment:
As I said before changing the image is not really ideal because Adverse is Adverse, challenge is challenge. You can't generalize it. 
`BattleStartAdverse`
![BattleStartAdverse](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/56174894/c1075664-6958-4979-a43e-cc74582efa26)
`BattleStartRaid`
![BattleStartRaid](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/56174894/919f2d61-417a-4764-9d6f-4e2fe97ebfa1)

Reply:
RaidDifficulty
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/102887808/91971ff3-fba5-4253-8b08-51c0e8bd9566)
NormalDifficulty
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/102887808/18a7a3b3-884e-4da7-845d-a86d11c7b187)

这里应该使用Raid的，一开始是看错`BattleStartRaid`和`BattleStartAdverse`了